### DISCUSSION TITLE
Installing to rpi

### DISCUSSION THREAD

**Discussion by ItsYaBoiDanie** [jef/streetmerchant#1223] [Help]
Hey so im trying to install it to my rpi running raspbian but its not working. I can install the pre req fine but once I get to the install part I followed the commands in the order and it did nothing. I looked it up in the issues thing but I dont get waht to do/ whats going on. I'm fairly new to Linux which does not help. So npm run start of course is not working. How do I install it? can someone help my dumbass out? Thanks!

**Comment by veetek22** [comment]
sorry to break it to you dont bother with pi , all youre gonna get is timeouts. Tried everything but rpi is just to slow to handle this many chromium threads.

**Comment by SAMLCSE** [comment]
It's bit of slow but I got this on my Raspberry Pi 4

**Comment by tingtingths** [comment]
> sorry to break it to you dont bother with pi , all youre gonna get is timeouts. Tried everything but rpi is just to slow to handle this many chromium threads.

if you have a spare ssd, use it with the pi 4. it helps a lot
